### PR TITLE
Updating getting started guide

### PR DESCRIPTION
**Pull request by DukeDavis12**
Updating getting started guide.

Signed-off-by: Davis Benny <davis.benny@intel.com>

### CODE REVIEW

File: docs/installation.md [hunk 1]
```diff
@@ -23,25 +23,27 @@ sudo apt-get remove docker docker-engine docker.io containerd runc
...
 ```
-curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
+curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo gpg --dearmor -o /usr/share/keyrings/docker-archive-keyring.gpg
 ```
 4 . Use the following command to set up the **stable** repository.
 ```
-sudo add-apt-repository \ "deb [arch=amd64] https://download.docker.com/linux/ubuntu \ $(lsb_release -cs) \ stable"
+echo \
+  "deb [arch=amd64 signed-by=/usr/share/keyrings/docker-archive-keyring.gpg] https://download.docker.com/linux/ubuntu \
+  $(lsb_release -cs) stable" | sudo tee /etc/apt/sources.list.d/docker.list > /dev/null
```
Comment (v-vasavi) [inline docs/installation.md:41]: Should we remove the $ symbols?

File: docs/installation.md [hunk 2]
```diff
@@ -23,25 +23,27 @@ sudo apt-get remove docker docker-engine docker.io containerd runc
...
 ```
 5 . Update the `apt` package index and install the Docker* Engine 18.09
```
Comment (v-vasavi) [inline docs/installation.md:43]: the * after Docker... can we add legend for that ?

File: docs/implementation-references/getting-started-guide.md [hunk 3]
```diff
@@ -4,6 +4,12 @@ not been implemented, and any aspect of this implementation is subject to change
 
 # Getting Started Guide
 
+<figure>
+  <img src="../../images/securedeviceonboard-icon-color.png" width="250" />
+</figure>
+
+***FDO (FIDO device onboard) provides a fast and more secure way to onboard any device to any device management system. A unique feature of FDO is the ability for the device owner to select the IoT platform at a late stage in the device life cycle. The secrets or configuration data may also be created or chosen at this late stage.***
```
Comment (v-vasavi) [inline docs/implementation-references/getting-started-guide.md]: can we change it to " to onboard "a" device "

File: docs/implementation-references/getting-started-guide.md [hunk 4]
```diff
@@ -4,6 +4,12 @@ not been implemented, and any aspect of this implementation is subject to change
...
+
 Quick walk through the E2E flow, so you can get started. Included in this guide:
```
Comment (v-vasavi) [inline docs/implementation-references/getting-started-guide.md]: Can we say - This document provides a quick walkthrough ?

File: docs/implementation-references/getting-started-guide.md [hunk 5]
```diff
@@ -16,71 +22,127 @@ Quick walk through the E2E flow, so you can get started. Included in this guide:
...
 ## Building FDO PRI Source
 
-!!! NOTE
-    For the instructions in this document, `<fdo-pri-src>` refers to the path of the FDO PRI folder 'pri-fidoiot'.
+!!! NOTES
+      - Check the [System Requirements](https://secure-device-onboard.github.io/docs-fidoiot/latest/installation/#system-requirements)
+      - If working behind a proxy,ensure to [set proper proxy](https://github.com/secure-device-onboard/pri-fidoiot/tree/master/component-samples/demo#configuring-proxies) variables.
+      - [Follow the steps](https://secure-device-onboard.github.io/docs-fidoiot/latest/installation/) to setup Docker* environment.
+      - [Read more](https://secure-device-onboard.github.io/docs-fidoiot/latest/installation/) about PRI source building.
+      - [Follow the steps](../../implementation-references/proxy-settings/) to set the right proxy settings.
```
Comment (v-vasavi) [inline docs/implementation-references/getting-started-guide.md]: I just realized there are two links to proxy settings and proxy variables.

File: docs/implementation-references/getting-started-guide.md [hunk 6]
```diff
@@ -16,71 +22,127 @@ Quick walk through the E2E flow, so you can get started. Included in this guide:
...
+```
+
+2.&nbsp; Build PRI-fidoiot:
 
 FDO PRI source can be built in two ways:
 
+
+!!! NOTE
+      For the instructions in this document, `<fdo-pri-src>` refers to the path of the FDO PRI folder 'pri-fidoiot'.
+      If you are not executing this command for the first time, ensure there are no docker instances of Owner, Manufacturer, RV running on the machine already.
```
Comment (v-vasavi) [inline docs/implementation-references/getting-started-guide.md]: Isnt this handled in the code now?

File: docs/installation.md (hunk 1 above)
Comment (DukeDavis12) [reply docs/installation.md:41]: We shouldn't.

File: docs/installation.md (hunk 2 above)
Comment (DukeDavis12) [reply docs/installation.md:43]: Adding legend at the end.

File: docs/implementation-references/getting-started-guide.md (hunk 3 above)
Comment (DukeDavis12) [reply docs/implementation-references/getting-started-guide.md]: Done.

File: docs/implementation-references/getting-started-guide.md (hunk 4 above)
Comment (DukeDavis12) [reply docs/implementation-references/getting-started-guide.md]: Done.

File: docs/implementation-references/getting-started-guide.md (hunk 6 above)
Comment (DukeDavis12) [reply docs/implementation-references/getting-started-guide.md]: Yes, Removing it.

File: docs/implementation-references/getting-started-guide.md (hunk 5 above)
Comment (DukeDavis12) [reply docs/implementation-references/getting-started-guide.md]: Removing the link for setting the right proxy.

Comment (v-vasavi) [reply docs/implementation-references/getting-started-guide.md]: Can we add it to this page instead? That is also a useful link right ?

Comment (DukeDavis12) [reply docs/implementation-references/getting-started-guide.md]: Okay.. We will retain it then.

**Review by v-vasavi — APPROVED** [review]

**Review by trbehera — APPROVED** [review]

**Pull request merged**